scripts/AbinsModules/AbinsTestHelpers.py

```diff
@@ -0,0 +1,58 @@
+import re
+import platform
+import mantid
+import AbinsConstants
+import os
+
+
+# Module with helper functions used to create tests.
+
+def version_as_tuple(string=None):
+    """
+    Calculates numerical representation of package version as a tuple.
+
+    :param string:  version of package in the format number1.number2.number3....numberN
+    :return: numerical representation of package version in the form  of tuple (number1, number2 ...., numberN)
+    """
+    if not isinstance(string, str):
+        raise ValueError("Version of package in the form of string is expected.")
+    if "." not in string:
+        raise ValueError("Invalid format of package version.")
+
+    try:
+        return tuple([int(i) for i in re.findall(r'\d+', string=string)])
+    except:
+        raise ValueError("Version of package couldn't be converted to number. (version=", string)
+
+
+def is_numpy_valid(string=None):
+    """
+
+    :param string: version of numpy to be checked in the format number1.number2.number3
+    :return: False if version of numpy  is valid otherwise True
+    """
+
+    return version_as_tuple(string=string) < version_as_tuple(AbinsConstants.NUMPY_VERSION_REQUIRED)
+
+
+def old_python():
+    """
+    Checks if Python i not too old
+    :return: True if it is too old otherwise False
+    """
+    return tuple([int(i) for i in re.findall(r'\d+', string=platform.python_version().replace(".", " "))]) < (2, 7)
+
+
+def find_file(filename=None):
+    """
+    Calculates path of filename with the testing data. Path is determined in the platform independent way.
+
+    :param filename: name of file to find
+    :return: full path for the file with the testing data
+    """
+    folders = mantid.config.getDataSearchDirs()
```

Comment:
Looks good to get operating system independence.

However you can avoid this code by using http://docs.mantidproject.org/nightly/api/python/mantid/api/FileFinderImpl.html?highlight=filefinder and or http://docs.mantidproject.org/nightly/api/python/mantid/api/FileProperty.html?highlight=fileproperty#mantid.api.FileProperty in any class where you load data. 

This also ensure that your loader will look in any of the folder specified by 'Managed properties'